cinch/roles/jenkins_master/tasks/plugins.yml

```diff
@@ -56,7 +56,7 @@
 
 - name: perform plugin install
   jenkins_plugin:
-    url: "{{ _jenkins_url }}"
+    url: "http://localhost:8080"
```

Comment:
If _jenkins_url is being generated improperly inside of ensure_up.yml, you should change it there instead of skipping over it here. If SSL is your only problem, then you can set validate_certs: false here.